src/tools/autocompletion/find_autocompletion_options.py

```diff
@@ -137,12 +139,56 @@ def complete_commands(start_of_input):
 # searches for options starting with start_of_input belonging to last_word
 # return: list of all possible arguments
 def complete_options():
-	global start_of_current_input, last_command
+	global last_command
+	completion_options = []
+	if last_command is not None:
+		completion_options.extend(complete_options_opt_array(last_command))
+	else:
+		completion_options.extend(complete_options_single())
+	return completion_options
+
+def complete_options_single():
+	global mount_point, start_of_current_input
+	completion_options = []
+	if start_of_current_input is not None:
+		start_of_word = start_of_current_input.replace('-', '')
+		len_of_word = len(start_of_current_input.strip())
+	k = kdb.KDB()
+	ks = kdb.KeySet()
+	k.get(ks, mount_point)
+	for key in ks:
+		path = str(key).split('/')
+		meta_opt = key.getMeta(name='opt')
+		if meta_opt and meta_opt.value.startswith('#'):
+			len_opts = int((meta_opt.value)[1:])
+			i = 0
+			while i <= len_opts:
+				i+=1
+				opt = key.getMeta(name='opt/#{}'.format(i))
+				opt_long = key.getMeta(name='opt/#{}/long'.format(i))
+				if start_of_current_input is None or len_of_word == 0:
+					if opt_long:
+						
```

Comment:
currently as a result I get -I and -R
so only opt for the last element in the list, when printing opt_long, it is always None. Could this be a problem with how my spec is mounted?

Reply:
You can always check if everything is as expected by using `kdb meta-ls` and `kdb meta-get`, e.g. `kdb meta-ls spec/tests/autocomplete/erm/force` or `kdb meta-get spec/tests/autocomplete/erm/force opt`